js/index.js

```diff
@@ -1,7 +1,78 @@
 // Iteration 1: Names and Input
-
+let hacker1 = "Phillip";
+console.log(`The driver's name is ${hacker1}`);
+let hacker2 = "Clara";
+console.log(`The navigator's name is ${hacker2}`);
 
 // Iteration 2: Conditionals
+if (hacker1.length > hacker2.length) {
+    console.log(`The driver has the longest name, it has ${hacker1.length} characters.`);
+  } else if (hacker1.length < hacker2.length) {
+    console.log(`It seems that the navigator has the longest name, it has ${hacker2.length} characters.`);
+  } else {
+    console.log(`Wow, you both have equally long names, ${hacker2.length} characters!.`);
+  }
+
+// Iteration 3: Loops
+let nameWithSpaces1 = "";
+
+for (let i=0; i < hacker1.length; i++){
+  nameWithSpaces1 += hacker1[i] + " ";
+}
+
+console.log(nameWithSpaces1.slice(0,13).toUpperCase());
```

Comment:
This 13 number that is added is hard coding. If you have a name that is 10 characters or 20 characters then this will not work. It only specifically works for names with 13 characters. Try to make it more dynamic. Hint: .length